tailwind.config.js

```diff
@@ -7,15 +7,15 @@
  * Tailwind Configuration.
  */
 
-var path = require("path");
+const path = require("path");
+const plugin = require('tailwindcss/plugin');
 
-var dir = path.resolve(__dirname, "src/plugins");
+const dir = path.resolve(__dirname, "src/plugins");
 
 module.exports = {
   // Our own prefix.
   prefix: "su-",
-  // Enable dark mode utility classes
-  darkMode: "media",
+  content: ["./dev/src/**/*.{html,js}"],
```

Comment:
I would like to avoid setting this in the base tw.config.js and use this file as a tailwind preset.

In the dev folder, for the tw.config.js file there, you should import this file as the preset and set the content path only there.